src/mux-analytics.brs

```diff
@@ -535,6 +584,122 @@ function muxAnalytics() as Object
     end if
   end sub
 
+  prototype.videoContentIndexChangeHandler = sub(contentIndex as integer)
+    if contentIndex > 0
+      m._addEventToQueue(m._createEvent("ended"))
+      m._endView(true)
+      m._startView(true)
+      m._triggerPlayEvent()
+      if m._Flag_atLeastOnePlayEventForContent = false
+        if m._viewTimeToFirstFrame = Invalid
+          if m._viewStartTimestamp <> Invalid and m._viewStartTimestamp <> 0
+            date = m._getDateTime()
+            now = 0# + date.AsSeconds() * 1000.0# + date.GetMilliseconds()
+            m._viewTimeToFirstFrame = now - m._viewStartTimestamp
```

Comment:
does `m._viewStartTimestamp` get updated with each new view, or is this going to make time to first frame equal the amount of time since the first piece of content was attempted?

Reply:
Yes, m._viewStartTimestamp will be updated with each new view. The state of playback does not change when the new content starts playing. So I had to assign the view start time for the new playback otherwise it not be calculated and always show 0 on the dashboard.